### DISCUSSION TITLE
Shimmer and DatePicker

### DISCUSSION THREAD

**Discussion by LoganBwDE** [microsoft/fluentui#25948] [General] [answered]
Hello,

on the Component Roadmap there is written that the components are to be addressed in Q2. Is there already an update on this?

**Comment by JustSlone** [comment, marked as answer]
Shimmer is now called Skeleton and is under development https://github.com/microsoft/fluentui/tree/master/packages/react-components/react-skeleton it is not available as a preview component yet. It will be available early in CY23-Q1. 
FYI: @tomi-msft 

DatePicker is a much more complex component, but we believe we have a plan to release a version later in CY23-Q1. 
FYI: @khmakoto 

Updated https://github.com/microsoft/fluentui/wiki/Fluent-UI-React-v9-Component-Roadmap to reflect the latest. We have a batch of components that are nearly stable that will likely move to stable early next year (holidays)

**Reply by khmakoto** [reply]
Hi @LoganBwDE, we indeed have a plan to release a version of `DatePicker` relatively soon. @sopranopillow and I are working on it and should have more details to share soon.
